pkg/controller/metrics/metrics.go

```diff
@@ -187,16 +187,25 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 
 				if cd.DeletionTimestamp == nil {
 					if !cd.Spec.Installed {
-						// Similarly for installing clusters we report the seconds since
-						// cluster was created. clusterdeployment_controller should set to 0
-						// once we know we've first observed install success, and then this
-						// metric should no longer be reported.
+						// For installing clusters we report the seconds since
+						// cluster was created. clusterdeployment_controller should delete this for any cluster
+						// that is installed or deleted.
 						MetricClusterDeploymentProvisionUnderwaySeconds.WithLabelValues(
 							cd.Name,
 							cd.Namespace,
 							GetClusterDeploymentType(&cd)).Set(
 							time.Since(cd.CreationTimestamp.Time).Seconds())
 					}
+				} else {
+
+					// For deprovisioning clusters we report the seconds since
+					// cluster was deleted. clusterdeployment_controller should delete this
+					// when removing the finalizer.
+					MetricClusterDeploymentDeprovisioningUnderwaySeconds.WithLabelValues(
+						cd.Name,
+						cd.Namespace,
+						GetClusterDeploymentType(&cd)).Set(
+						time.Since(cd.CreationTimestamp.Time).Seconds())
```

Comment:
Oops. Copypasta.
```suggestion
						time.Since(cd.DeletionTimestamp.Time).Seconds())
```
Fixed via #1981.